runtime/doc/eval.txt

```diff
@@ -2841,9 +2841,8 @@ diff_hlID({lnum}, {col})				*diff_hlID()*
 empty({expr})						*empty()*
 		Return the Number 1 if {expr} is empty, zero otherwise.
 		A |List| or |Dictionary| is empty when it does not have any
-		items.	A Number is empty when its value is zero.
-		For a long |List| this is much faster than comparing the
-		length with zero.
+		items.  A Number is empty when its value is zero.  Special 
+		variable is empty when it is |v:false| or |v:none|.
```

Comment:
@ZyX-I Since `v:none` was [removed](https://github.com/neovim/neovim/pull/4131/commits/b6c48798512f0ae2b50effc899da9ed2c8a5e029) it should not be mentioned here.